Src/TestTypeFoundation/TypeWithPseudoFactoryMethod.cs

```diff
@@ -1,9 +1,9 @@
 ﻿namespace TestTypeFoundation
 {
     /// <summary>
-    /// This type contains a method that returns an instance of the same type 
-    /// so that it looks like a Factory Method. In fact, it is not because it 
-    /// receives a parameter of the same type, so using it as a factory inevitably 
+    /// This type contains a method that returns an instance of the same type
+    /// so that it looks like a Factory Method. In fact, it is not because it
+    /// receives a parameter of the same type, so using it as a factory inevitably
```

Comment:
Nice! I have Sublime Text do this for me automatically on save, but happy to see this being applied here.